view/utils.js

```diff
@@ -61,4 +61,6 @@ function parseQueryParams(params) {
         acc[key] = value ?? false;
         return acc;
     }, {});
-}
\ No newline at end of file
+}
+
+let tob = (arr) => new Blob([arr], {type: 'application/octet-stream'});
```

Comment:
Please make this function name more explicit, like `array_to_binary` or `to_binary`